dm/templates/vpn/vpn.py

```diff
@@ -21,19 +21,34 @@ def generate_config(context):
     properties = context.properties
     project_id = properties.get('project', context.env['project'])
 
-    network = generate_network_url(
+    network = context.properties.get('networkURL', generate_network_uri(
         project_id,
-        context.properties['network']
-    )
+        context.properties.get('networkName','')
+    ))
     target_vpn_gateway = context.env['name'] + '-tvpng'
-    static_ip = context.env['name'] + '-ip'
     esp_rule = context.env['name'] + '-esp-rule'
     udp_500_rule = context.env['name'] + '-udp-500-rule'
     udp_4500_rule = context.env['name'] + '-udp-4500-rule'
     vpn_tunnel = context.env['name'] + '-vpn'
     router_vpn_binding = context.env['name'] + '-router-vpn-binding'
+    resources = []
+    static_ip = ''
+    ip_address = ''
```

Comment:
I don't think we need to define these empty strings upfront.

Reply:
ip_address can be removed.
static_ip is necessary because we need it in output.
resources is necessary because we want to use "append" and "extend" in the following codes.

Reply:
Ok please remove ip_address.